game_event/views.py

```diff
@@ -32,7 +32,10 @@ def process_game_event_form(request):
         ball_game = request.POST.get('ball_game')
 
         try:
-            GameEvent.create(time, level_of_game, min_number_of_players, max_number_of_players, court, ball_game)
+            player = Player.objects.get(user=request.user)
+            event = GameEvent.create(time, level_of_game, min_number_of_players,
+                                     max_number_of_players, court, ball_game)
+            GameEventPlayer(game_event=event, player=player, ball_responsible=False).save()
```

Comment:
We should use here `GameEventPlayer.objects.create(...)` instead. No? This way I think we have some more built-in validations.